tmp/lanelet2_extension/lib/utilities.cpp

```diff
@@ -501,12 +501,28 @@ lanelet::ConstLanelets getExpandedLanelets(
 }
 
 void overwriteLaneletsCenterline(
-  lanelet::LaneletMapPtr lanelet_map, const double resolution, const bool force_overwrite)
+  lanelet::LaneletMapPtr lanelet_map, const double resolution, const bool use_waypoints,
```

Comment:
Just as a comment, since the third argument of the function has changed it's meaning, we should make sure that any other code that uses `force_overwrite` argument should be merged at the same time as this PR.

Reply:
@takayuki5168 Have you checked if the codes in autoware.universe and autoware_tools are fine with the modification?

Reply:
Thank you for the review.
Yes, I've already checked. The following PR is the change for the universe.
https://github.com/autowarefoundation/autoware.universe/pull/7480/files